clients/java/src/main/java/io/camunda/zeebe/client/impl/response/MatchedDecisionRuleImpl.java

```diff
@@ -19,17 +19,25 @@
 import io.camunda.zeebe.client.api.JsonMapper;
 import io.camunda.zeebe.client.api.response.EvaluatedDecisionOutput;
 import io.camunda.zeebe.client.api.response.MatchedDecisionRule;
+import io.camunda.zeebe.client.protocol.rest.MatchedDecisionRuleItem;
 import io.camunda.zeebe.gateway.protocol.GatewayOuterClass;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class MatchedDecisionRuleImpl implements MatchedDecisionRule {
 
-  @JsonIgnore private final JsonMapper jsonMapper;
+  @JsonIgnore private JsonMapper jsonMapper;
```

Comment:
❓ Do we need to add the JsonMapper as well when we instantiate this class from REST responses? Otherwise we cannot call `toJson` on it and I'm not sure if that will become an issue somewhere 🤔

Reply:
Where we call `toJson` ?

Reply:
I actually don't know 😅 I was just flagging this since the mapper is used in that method. If that method is called and we're using the REST communication, the `toJson` call will run into an exception, I guess.

Reply:
We are not using the method with the mapper for REST, if I'm not wrong

Reply:
Do you have an idea where the `toJson` is called?

Reply:
No, but I don't understand why you think it is a problem 👀

Reply:
My concern is that the jsonMapper in this class is null in REST use cases. If, in those REST use cases, the toJson method of this class is called from anywhere (serialization of the response or anything), that call will fail. I can have a closer look at the class later on a proper screen, I'm just looking at this from my mobile currently 😄

Reply:
Synced offline: we'll pass in the `jsonMapper` for REST as well since it's used in the `toString` method